src/term/mod.rs

```diff
@@ -950,12 +954,16 @@ impl ansi::Handler for Term {
 
     #[inline]
     fn save_cursor_position(&mut self) {
-        err_println!("[unimplemented] save_cursor_position");
+        self.saved_cursor = Some(self.cursor);
     }
 
     #[inline]
     fn restore_cursor_position(&mut self) {
-        err_println!("[unimplemented] restore_cursor_position");
+        if let Some(pos) = self.saved_cursor {
+            self.cursor = pos;
```

Comment:
SIGWINCH would be for programs running inside Alacritty. The terminal is essentially responsible for generating these signals (indirectly): https://github.com/jwilm/alacritty/blob/master/src/tty.rs#L325.

It may not have been an issue in the case you tested, but there's certainly the possibility for a panic when this happens.